packages/components/constraints/package.json

```diff
@@ -36,6 +36,6 @@
     "react": ">= 16.8.0"
   },
   "readme": {
-    "componentPath": "./src/horizontal/horizontal.js"
+    "componentPath": "./src/horizontal/horizontal.tsx"
```

Comment:
I'm not sure about this part.

Reply:
It's for the readme generation. You should also run it